backend/src/data/seed-data/comment.data.ts

```diff
@@ -1,4 +1,144 @@
 export const comments = [
+  {
+    id: '23a94b8b-850e-4dc1-8eb9-788f8cd152ac',
+    text: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Quidem, aut.',
+    authorId: '9a25ad37-a647-4c1a-897f-59dbf275765d',
+    pageId: 'f8cd8ff3-d1de-47a5-820e-73a58368b22e',
+    parentCommentId: null,
```

Comment:
```suggestion
    text: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Quidem, aut.',
    authorId: '9a25ad37-a647-4c1a-897f-59dbf275765d',
    pageId: 'f8cd8ff3-d1de-47a5-820e-73a58368b22e',
    parentCommentId: null,
```

id is added by db by default